libs/hwloc/Makefile

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (C) 2012-2018 OpenWrt.org
```

Comment:
no, the date is wrong, and unless you have copyright assignment on file with openwrt.org, this is not valid. Either drop this entirely, or use your own name.